projects/gateway2/proxy_syncer/upstreams.go

```diff
@@ -25,10 +25,15 @@ import (
 )
 
 type uccWithCluster struct {
```

Comment:
nit `uccWithResources`?

Reply:
I understand that the struct is cluster specific, but it may have additional resources that are driven by the options on the Upstream/Backend. Framed like that, I'm hesitant to remove "cluster" from the type name.